p2p/libp2p/netMessenger.go

```diff
@@ -313,11 +340,16 @@ func (netMes *networkMessenger) printLogs() {
 	log.Info("listening on addresses", addresses...)
 
 	go netMes.printLogsStats()
+	go netMes.checkExternalLoggers()
 }
 
 func (netMes *networkMessenger) printLogsStats() {
 	for {
-		time.Sleep(secondsBetweenPeerPrints * time.Second)
+		select {
+		case <-netMes.ctx.Done():
+			return
+		case <-time.After(secondsBetweenPeerPrints * time.Second):
```

Comment:
make directly in const to be equal with value * time.Second

Reply:
done